### PR TITLE
Style table

### PR DESCRIPTION
**Pull request by tnyandoro**
Styling  of the  add button
Styling of the title and author
Adding table and styling it
Stying of the container of the book

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,16 +7,37 @@
...
+          <th scope="col">Title</th>
+          <th scope="col">Author</th>
+          <th scope="col"></th>
+        </tr>
+      </thead>
+      <tbody>
+      </tbody>
+    </table>
     <br>
     <form id="book-form">
```
Comment (mricanho) [inline index.html]: - [ ] The width of this form must be shorter, remember that you need to follow the original layout:

Your layout:

image

Original layout:

awesome_books_core_elements

File: index.html [hunk 2]
```diff
@@ -7,16 +7,37 @@
...
       <label for="title" class="form-label">Title</label>
```
Comment (mricanho) [inline index.html]: - [ ] The Add button must be placed on the right side, not in the left side

Your layout:

image

Original layout:

awesome_books_core_elements

File: index.html [hunk 3]
```diff
@@ -7,16 +7,37 @@
...
   <div class="container">
-    <h1 class="display"></h1>
-    <div class="books">
+    <div class="display">
+      <h1>All Awesome Books</h1>
     </div>
+    <table class="table table-striped books">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
```
Comment (mricanho) [inline index.html]: - [ ] This column it's not in the original layout, you can erase it

Your layout:

image

Original layout:

awesome_books_core_elements

File: index.html [hunk 4]
```diff
@@ -7,16 +7,37 @@
...
+          <th scope="col">Title</th>
+          <th scope="col">Author</th>
```
Comment (mricanho) [inline index.html]: - [ ] Try to assemble the Title and the author in the same row connected by a "by", as the original layout, you don't need the head titles too

Your layout:

image

Original layout:

awesome_books_core_elements

File: index.html [hunk 5]
```diff
@@ -7,16 +7,37 @@
...
 
 <body>
-  <h1 class="catch">Awesome Books</h1>
+<div>
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">Programming</a>
+  <a href="#contact">Space Sciences</a>
+  <a href="#about">About</a>
+</div>
```
Comment (mricanho) [inline index.html]: - [ ] Remove this nav, it's not necessary, as you can see in the original layout

Your layout:

image

Original layout:

awesome_books_core_elements

File: index.html (hunk 1 above)
Comment (mricanho) [inline index.html]: - [ ] Add a title "Add a new book" as the original layout, on top of the form, if you feel super savage, add an "hr" on top of that title.

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @tnyandoro @erezfree29, 

Good job so far! You are following the gitflow and your commit messages are good too, using bootstrap it's a good call🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Check the inline comments, but basically, you need to follow the layout given to you 

_Check the comments under the review._

## Optional suggestions

- [ ]  [SUPER OPTIONAL] You have a bug every time you want to add a book, it doesn't appear on the list until you refresh the page, double-check your submit button.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

you can do it

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tnyandoro** [comment]
Changes 
removed the navbar
set the button the add book button to the right
Resized the form
Checked no bug for the listed book after added no need to refresh for it to appear on the DOM.

File: index.html [hunk 6]
```diff
@@ -7,24 +7,45 @@
...
 
 <body>
-  <h1 class="catch">Awesome Books</h1>
+  <div>
+  </div>
   <div class="container">
-    <h1 class="display"></h1>
-    <div class="books">
+    <div class="display">
+      <h1>All Awesome Books</h1>
```
Comment (smcommits) [inline index.html]: - [ ] **In implementing the changes, the alignment of your "All Awesome Books" and "Add Book" headings is mistakenly disturbed. Please fix it to match the wireframe**

Cloned Design|Original Design
:------------------:|:-------------------:
image|image

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @tnyandoro, 

### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by tnyandoro** [comment]
We have implemented the changes, the alignment of our "All Awesome Books" and "Add Book" headings was mistakenly disturbed. It has been  fixe  to match the wireframe

**Review by ZawadiSumba66 — APPROVED** [review]
#### Hi @erezfree29 👋 👋 , 

## STATUS: APPROVED ✔️ ✔️ 

All the changes requested by the previous reviewers are now reflecting!! There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**